File: PimSuporte/app/src/main/java/com/pim/pimsuporte/models/Usuario.java
package com.pim.pimsuporte.models;
import java.io.Serializable;
import com.google.gson.annotations.SerializedName;

public class Usuario implements Serializable {

    // JSON: "id_usuario" -> Java: idUsuario
    @SerializedName("id_usuario")
    private int idUsuario;

    // JSON: "fk_tipo_usuario" -> Java: fkTipoUsuario
    @SerializedName("fk_tipo_usuario")
    private int fkTipoUsuario;

    // JSON: "tipo_usuario" -> Java: tipoUsuario
    @SerializedName("tipo_usuario")
    private String tipoUsuario;

    // JSON: "nome" -> Java: nome
    @SerializedName("nome")
    private String nome;

    // JSON: "email" -> Java: email
    @SerializedName("email")
    private String email;

    public Usuario() {
    }

    public int getIdUsuario() {
        return idUsuario;
    }

    public int getFkTipoUsuario() {
        return fkTipoUsuario;
    }

    public String getTipoUsuario() {
        return tipoUsuario;
    }

    public String getNome() {
        return nome;
    }

    public String getEmail() {
        return email;
    }


    public void setIdUsuario(int idUsuario) {
        this.idUsuario = idUsuario;
    }

    public void setFkTipoUsuario(int fkTipoUsuario) {
        this.fkTipoUsuario = fkTipoUsuario;
    }

    public void setTipoUsuario(String tipoUsuario) {
        this.tipoUsuario = tipoUsuario;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}

File: PIM4-Desktop/Models/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Text.Json.Serialization;

namespace PIM4_Desktop.Models
{
    public class Usuario
    {
        [JsonPropertyName("id_usuario")]
        public int IdUsuario { get; set; }

        [JsonPropertyName("fk_tipo_usuario")]
        public int FkTipoUsuario { get; set; }

        [JsonPropertyName("tipo_usuario")]
        public string TipoUsuarioNome { get; set; }

        [JsonPropertyName("nome")]
        public string Nome { get; set; }

        [JsonPropertyName("email")]
        public string Email { get; set; }
    }
}
